<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vegetablesClass</class>
 <widget class="QMainWindow" name="vegetablesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vegetables</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QListWidget" name="searched_vegetables"/>
      </item>
      <item row="0" column="0">
       <widget class="QListWidget" name="repo_families"/>
      </item>
      <item row="0" column="1">
       <widget class="QListWidget" name="repo_vegetables_from_family"/>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="vegetable_name"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="Search">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vegetables.qrc"/>
 </resources>
 <connections/>
</ui>
